### PR TITLE
Integrate HSReplay.net with MailChimp

### PR DESCRIPTION
**Pull request by joolean**

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -45,6 +46,7 @@ hsarchetypes = {file = "https://github.com/HearthSim/hsarchetypes/archive/master
 django-bouncy = "*"
 unidecode = "*"
 pipenv = "*"
+requests-mock = "*"
```
Comment (beheh) [inline Pipfile]: This should probably only be a development dependency.

File: hsreplaynet/utils/mailchimp.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+_client.lists.members.tags = ListMemberTags(_client)
+
+
+def get_mailchimp_client():
+	"""Return a configured instance of the MailChimp API client."""
+
+	return _client
+
+
+def find_best_email_for_user(user):
```
Comment (beheh) [inline hsreplaynet/utils/mailchimp.py]: We should probably only be returning email addresses that the user has verified.

Comment (joolean) [reply hsreplaynet/utils/mailchimp.py]: I talked about this with Tiago - he wants them all.

File: Pipfile (hunk 1 above)
Comment (joolean) [reply Pipfile]: Oops! Will fix.

**Pull request merged**